### PR TITLE
Create base RESTFUL

### PR DESCRIPTION
**Pull request by esdrasedu**

### CODE REVIEW

**Comment by lucca65** [comment]
+1

**Comment by leorog** [comment]
+1

File: lib/kiik/rest/create.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          when 200
+            build(JSON.parse(response.body))
+          when 422
+            raise KiikError.new(JSON.parse(response.body))
+          else
+            raise Error.new(response.message)
+          end
+        end
+
+        def build(data, error = nil)
```
Comment (esdrasedu) [inline lib/kiik/rest/create.rb]: this method should be of `Kiik::Resource` my bad :-1:

File: lib/kiik/customer.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 module Kiik
-  class Customer
+  class Customer < Resource
+    include Kiik::Rest::Create
+
+    attr_accessor :phone, :name, :email, :metadata, :description, :cards
```
Comment (jonatandahora) [inline lib/kiik/customer.rb]: document missing

**Comment by jonatandahora** [comment]
+1

**Pull request merged**